src/main/java/uk/gov/hmcts/reform/feature/exception/PendingMigrationScriptException.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.reform.feature.exception;
+
+public class PendingMigrationScriptException extends RuntimeException {
```

Comment:
Do we need to extend UnknowErrorCodeException? Else do we get exceptions in app insights ?

Reply:
I've [disabled](https://github.com/hmcts/feature-toggle-api/blob/master/infrastructure/main.tf#L23-L24) those 😆